4-css-grid/styles/styles.css

```
@@ -0,0 +1,37 @@
+.A {
```

use lowercase classnames. 
```suggestion
.a {
```